build.rs

```diff
@@ -101,4 +112,8 @@ fn mm_version() -> String {
     version
 }
 
-fn main() { mm_version(); }
+fn main() {
+    // This allows build script to run even if no source code files change as rerun-if-changed checks for a file that doesn't exist
+    println!("cargo:rerun-if-changed={}", path2s(rabs("NON_EXISTING_FILE")));
```

Comment:
Could you please clarify why path2s and rabs are required here?

Reply:
I guess it's not needed since it checks for a file that doesn't exist. Will fix it